docs/software/webserver_stack.md

```diff
@@ -469,7 +469,7 @@ LLSP is a variation of the popular open source [LAMP web stack](#lamp-web-stack)
 
 ## WebServers
 
-### Apache2
+### Apache {: #apache }
```

Comment:
`{: #apache }` is not required here, the heading text in lower case letters is used automatically.